pay-api/src/pay_api/services/payment_account.py

```diff
@@ -652,6 +664,38 @@ def find_by_id(cls, account_id: int):
         account._dao = PaymentAccountModel.find_by_id(account_id)  # pylint: disable=protected-access
         return account
 
+    @classmethod
+    def find_eft_accounts(cls, eft_accounts_search: EFTAccountsSearch):
+        """Find EFT accounts."""
+        query = db.session.query(PaymentAccountModel) \
+            .outerjoin(EFTShortnamesModel, PaymentAccountModel.auth_account_id == EFTShortnamesModel.auth_account_id) \
+            .filter(PaymentAccountModel.payment_method == PaymentMethod.EFT.value,
+                    PaymentAccountModel.eft_enable.is_(True))
+
+        if eft_accounts_search.state == EFTShortnameState.UNLINKED.value:
+            query = query.filter(EFTShortnamesModel.auth_account_id.is_(None))
+        elif eft_accounts_search.state == EFTShortnameState.LINKED.value:
+            query = query.filter(EFTShortnamesModel.auth_account_id.isnot(None))
+
+        query = query.order_by(PaymentAccountModel.id)
+        pagination = query.paginate(per_page=eft_accounts_search.limit, page=eft_accounts_search.page)
+
+        total = pagination.total
```

Comment:
`'total': pagination.total`

`eft_accounts_list = [PaymentAccountSearchModel.from_row(eft_account) for eft_account in pagination.items]`